src/client/ducks/session/index.js

```diff
@@ -3,8 +3,7 @@
 import { pagination } from 'ducks/pagination'
 import { orderBy, DIRECTION } from 'ducks/order'
 import { combineReducers } from 'redux'
-import { target } from 'ducks/filters'
-import { PullRequestSource, DEVELOPER_PERSONA, PullRequestOrderFields } from 'universal/constants'
+import { DEVELOPER_PERSONA, PullRequestOrderFields } from 'universal/constants'
```

Comment:
We need remove `DEVELOPER_PERSONA` and personas constants at all. But that not related to your changes, just noticed :)